src/zh/mycomic/src/eu/kanade/tachiyomi/extension/zh/mycomic/MyComic.kt

```diff
@@ -0,0 +1,221 @@
+    override val lang: String
+        get() = "zh"
+    override val name: String
+        get() = "MyComic"
+    override val supportsLatest: Boolean
+        get() = true
+
+    override fun headersBuilder() = super.headersBuilder().add("referer", baseUrl)
+    private val json by injectLazy<Json>()
+    private val preferences by getPreferencesLazy()
+    private val requestUrl: String
+        get() = if (preferences.getString(PREF_KEY_LANG, "") == "zh-hans") {
+            "$baseUrl/cn"
+        } else {
+            baseUrl
+        }
+
+    override fun chapterListParse(response: Response): List<SChapter> {
+        val doc = response.asJsoup()
+        val data = doc.select("div[data-flux-card] + div div[x-data]").attr("x-data")
+        val chapters =
+            data.substringAfter("chapters:").substringBefore("\n").trim().removeSuffix(",")
+        return json.decodeFromString<JsonArray>(chapters).map {
+            SChapter.create().apply {
+                name = it.jsonObject["title"]!!.jsonPrimitive.content
+                // Since the images included in the chapter do not distinguish between Traditional and Simplified Chinese, the default URL will be used uniformly here.
+                // Additionally, using different URLs would create more issues, so it's best to keep the URL consistent.
+                url = "/chapters/${it.jsonObject["id"]!!.jsonPrimitive.content}"
+            }
+        }
+    }
+
+    override fun chapterFromElement(element: Element) = throw UnsupportedOperationException()
+
+    override fun chapterListSelector() = throw UnsupportedOperationException()
+
+    override fun imageUrlParse(document: Document) = throw UnsupportedOperationException()
+
+    override fun latestUpdatesFromElement(element: Element) = searchMangaFromElement(element)
+
+    override fun latestUpdatesNextPageSelector() = searchMangaNextPageSelector()
+
+    override fun latestUpdatesRequest(page: Int) =
+        searchMangaRequest(page, "", FilterList(latestUpdateFilter))
+
+    override fun latestUpdatesSelector() = searchMangaSelector()
+
+    override fun mangaDetailsParse(document: Document): SManga {
+        val detailElement = document.select("div[data-flux-card]")
+        return SManga.create().apply {
+            title = detailElement.select("div[data-flux-heading]").text().ifEmpty { title }
+            thumbnail_url = detailElement.select("img.object-cover").attr("src")
+            status = detailElement.select("div[data-flux-badge]").text().let {
+                when (it) {
+                    "连载中", "連載中" -> SManga.ONGOING
+                    "已完结", "已完結" -> SManga.COMPLETED
+                    else -> SManga.UNKNOWN
+                }
+            }
+            detailElement.select("div[data-flux-badge] + div").let {
+                author = it.select(":first-child a").text()
+                genre = it.select(":nth-child(3) a").joinToString { e -> e.text() }
```

Comment:
`e` gives no additional context.
```suggestion
                genre = it.select(":nth-child(3) a").joinToString { it.text() }
```

Reply:
`it` has already been used in the outer scope, so to prevent variable shadowing, the variable name is explicitly defined within the closure.

Reply:
Would setting the name explicitly for `.let` help?